exercises/scale-generator/tests/scale-generator.rs

```diff
@@ -0,0 +1,179 @@
+//! Tests for scale-generator
+//!
+//! Generated by [script][script] using [canonical data][canonical-data]
+//!
+//! [script]: https://github.com/exercism/rust/blob/master/bin/init_exercise.py
+//! [canonical-data]: https://raw.githubusercontent.com/exercism/problem-specifications/master/exercises/scale-generator/canonical_data.json
```

Comment:
oh, whoops, is the generator emitting canonical_data.json as the filename? Should be canonical-data.json

Reply:
Oh, good catch. Created https://github.com/exercism/rust/issues/669 for this.

Reply:
https://www.fuckbook.com/es/account/premium?